### PR TITLE
Added the web_app

### PR DESCRIPTION
**Pull request by JTarakRam**
Added the streamlit web app, where The users can directly test the auto label, with just running the streamlit run app.py

Note this is not yet deployed so it runs only in local host.

### CODE REVIEW

**Comment by JTarakRam** [comment]
I Have converted the standardvscotprompt.png into jpg and changed the code of it, now you can see the image, 

note - Please replace my name with the user name :)

File: docs/guide/accuracy/chain-of-thought.md [hunk 1]
```diff
@@ -1,7 +1,4 @@
-<figure markdown>
-  ![Chain-of-Thought prompting](/assets/standardvscotprompt.png){ width="600" }
-  <figcaption>Chain of Thought Prompting (Wei et al)</figcaption>
-</figure>
+![Chain-of-Thought-Prompting](https://raw.githubusercontent.com/JTarakRam/autolabel/main/docs/assets/standardvscotprompt.jpg)
```
Comment (nihit) [inline docs/guide/accuracy/chain-of-thought.md:1]: thanks for suggesting it! I think the chain of thought doc works correctly and as expected currently though? Here's the URL: https://docs.refuel.ai/guide/accuracy/chain-of-thought/

Further, this proposed fix references a URL in `JTarakRam/autolabel` fork

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,114 @@
+aiohttp==3.8.4
```
Comment (nihit) [inline requirements.txt:1]: we maintain requirements in pyproject.toml file

**Review by nihit — COMMENTED** [review]
Thanks Tarak for proposing this change. We are planning to add a lightweight UI support in Autolabel very soon as discussed in our Discord. You're very welcome to continue to use this in your fork in the meantime though!

Also take a look at our contributing guidelines (https://github.com/refuel-ai/autolabel/blob/main/CONTRIBUTING.md) or open Github issues (https://github.com/refuel-ai/autolabel/issues) for bug fixes/enhancements that might be good candidates for immediate contributions to Autolabel!

File: requirements.txt (hunk 2 above)
Comment (JTarakRam) [reply requirements.txt:1]: Oh !, My bad  I used to do every time requirements.txt , Haha

File: docs/guide/accuracy/chain-of-thought.md (hunk 1 above)
Comment (JTarakRam) [reply docs/guide/accuracy/chain-of-thought.md:1]: Yeah, It is working perfectly Fine !

**Pull request closed without merging**